src/track/keyutils.cpp

```diff
@@ -7,9 +7,6 @@
 #include "track/keyutils.h"
 #include "util/math.h"
 
-#define MUSIC_FLAT_UTF8  "\xe299ad"
```

Comment:
These constants are actually broken and produce invalid traditional key notation file tags.